Automattic/pocket-casts-ios
# Bookmarks: Wrap the episode detail view in a scrollview

In preparation for adding the bookmark tabs to the episode detail this adds a container scroll view around existing episode detail.

This scroll view will be used for horizontal scrolling to the bookmarks tab, but in this PR it doesn't do anything. 

## Screenshots

### iPhone Before / After
| Before | After | 
|:---:|:---:|
| <img src="https://github.com/Automattic/pocket-casts-ios/assets/793774/5e58740c-65f0-4b94-a571-2c9a83f7d4c7" width="320" />| <img src="https://github.com/Automattic/pocket-casts-ios/assets/793774/e0b09036-134c-4b15-8fa4-2b0f6f87b5e6" width="320" />|

### iPad Before / After
| Before | After | 
|:---:|:---:|
| <img width="320" src="https://github.com/Automattic/pocket-casts-ios/assets/793774/75a03466-8a87-4c8d-aab7-6e07621943ce">| <img width="320" src="https://github.com/Automattic/pocket-casts-ios/assets/793774/50b841d2-c58e-4e78-a3da-f8c44e82b6c2"> |


## To test
1. Launch the app
2. Tap on the Podcasts tab
3. Tap on any podcast
4. Tap on any episode
5. ✅ Verify the view looks and functions the same as it did before
6. ✅ Verify there are no auto layout warnings in console

## Checklist
